osgi/karaf/features/src/main/resources/features.xml

```diff
@@ -314,7 +314,7 @@
         <bundle start-level="35">mvn:org.apache.cxf/cxf-rt-rs-service-description-swagger-ui/${project.version}</bundle>
         <bundle start-level="40">mvn:org.apache.cxf/cxf-rt-rs-service-description-swagger/${project.version}</bundle>
         <bundle start-level="35" dependency="true">mvn:com.fasterxml.jackson.datatype/jackson-datatype-jsr310/${cxf.jackson.version}</bundle>
-        <bundle start-level="10" dependency="true">mvn:jakarta.validation/jakarta.validation-api/${cxf.validation.api.version}</bundle>
+        <bundle start-level="10" dependency="true">mvn:javax.validation/validation-api/1.1.0.Final</bundle>
```

Comment:
@amarkevich It would be better to override manifest instead, 
```
<bundle start-level="10" dependency="true">mvn:jakarta.validation/jakarta.validation-api/${cxf.validation.api.version}$overwrite=merge&amp;javax.validation.constraints;version="[1.1,3)"</bundle>
```
I have sent the PR against 1.6.x branch as well (https://github.com/swagger-api/swagger-core/pull/3497) but it may take some time.

Reply:
added

Reply:
:+1: Thanks!